plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/image/block.json

```diff
@@ -24,6 +24,7 @@
 			"clientNavigation": true
 		},
 		"html": false,
+		"layout": true,
```

Comment:
AFAIK `layout: true` is not enough to display [layout controls](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#layout).

You should provide `default` in order to set the layout `flex`. That could simplify `style.scss` changes and manual `atteibutes['layout']` mapping as it would be handled natively by Core.